src/main/java/blusunrize/immersiveengineering/common/crafting/ArcRecyclingThreadHandler.java

```diff
@@ -41,6 +44,7 @@ public ArcRecyclingThreadHandler(Collection<IRecipe<?>> allRecipes)
 		this.recipeList = allRecipes.stream()
 				.filter(ArcFurnaceRecipe.assembleRecyclingFilter())
 				.collect(Collectors.toList());
+		runningHandler = this;
```

Comment:
Could have kept style here and put a `this.` before it

Reply:
No, because `runningHandler` is static. I can put `ArcRecyclingThreadHandler.` before it if you want.

Reply:
That checks out, I'm a dumb dumb!